app/models/paper.rb

```diff
@@ -2,4 +2,7 @@ class Paper < ApplicationRecord
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references
+
+  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
```

Comment:
I see. It might silently fail if this isn't here? Like the authors creation will fail, but the paper will save or something.